ingest/validation/validate_metadata.py

```diff
@@ -211,9 +213,15 @@ def retrieve_ols_term(self, ontology_urls, term, property_name):
         try:
             ontology_shortname, term_id = re.split("[_:]", term)
         except (ValueError, TypeError):
-            raise ValueError(
-                f'{property_name}: Could not parse provided ontology id, "{term}"'
-            )
+            msg = f'{property_name}: Could not parse provided ontology id, "{term}".'
+            if type == "array":
+                if "|" not in term:
+                    msg += (
+                        f" There is only one array value, for ontology id, '{term}.' "
```

Comment:
I like that we're specifically calling this out now - a good enhancement.

Reply:
I don't understand how this check is operating.  this method is supposed to only handle single terms, right?  So if a term with a pipe got passed to this method, doesn't that mean our upstream parsing failed to recognize it as an array?

Reply:
We actually allow single value arrays. This is shown in the valid array test file [here](https://github.com/broadinstitute/scp-ingest-pipeline/blob/2bcddddf4ecc6a508be47a198d9751316cbe30c9/tests/data/annotation/metadata/convention/valid_array_v2.1.2.txt#L7). This check is here due to this issue stated in [this comment](https://github.com/broadinstitute/scp-ingest-pipeline/pull/209/files#r591691638)